Work around a ports bug wrt symlinking manpages in different
categories, by copying the manpage instead.

--- Makefile.am.orig	2013-01-30 11:40:06.000000000 +0900
+++ Makefile.am	2014-02-08 01:46:11.000000000 +0900
@@ -118,7 +118,7 @@
 install-data-hook:
 	cd $(DESTDIR)$(mandir)/man1 && $(RM) play.1 && $(LN_S) sox.1 play.1
 	cd $(DESTDIR)$(mandir)/man1 && $(RM) rec.1 && $(LN_S) sox.1 rec.1
-	cd $(DESTDIR)$(mandir)/man7 && $(RM) soxeffect.7 && $(LN_S) ../man1/sox.1 soxeffect.7
+	cd $(DESTDIR)$(mandir)/man7 && $(RM) soxeffect.7 && cp ../man1/sox.1 soxeffect.7
 
 uninstall-hook:
 	$(RM) $(DESTDIR)$(mandir)/man1/play.1
